packages/chart/src/data/initial-state.js

```diff
@@ -13,6 +13,7 @@ export default {
   animate: false,
   lineDatapointStyle: 'hover',
   lineDatapointColor: 'Same as Line',
+  enlargeIsolatedDots: true,
```

Comment:
this will also require a migration to make the legacy charts have this defaulted to on.